### DISCUSSION TITLE
Combats contre monstres

### DISCUSSION THREAD

**Discussion by Pietagorh** [Crownicles/Crownicles#601] [Nouvelles idées]
Ajout de micro-combats dans les évents contre des monstres. Les combats seraient automatisés et dépendraient donc des stats du joueur.
il est peu probable qu'un système de réaction existe pour controler ces combats.

**Comment by niqore** [comment]
Petite idée d'interface vite fait
![interface rapport monstres](https://user-images.githubusercontent.com/18489162/136696405-be58a8cb-dcb9-4359-bbeb-34642eac4eec.png)
